src/clarity/main.scss

```diff
@@ -1,9 +1,13 @@
+
 /*
```

Comment:
I just noticed this isn't loud, which means the copyright probably isn't in the minified CSS. Do you mind fixing this as part the your commit?